atd-toolbox/s3_restore_cr3_pdfs/s3_restore_cr3_pdfs.py

```diff
@@ -0,0 +1,268 @@
+from __future__ import print_function
```

Comment:
@frankhereford are you running this in a python 2.x environment? i'm curious why this import is needed.

Reply:
Good call; It's actually not needed. I picked up that line from the page I mentioned on line 15, and it isn't needed on the 3.9.5 version that I'm using to develop this. 

I'd like to take your feedback that you've given and update the program to implement it (including striking that line) - it's really helpful direction and I'm grateful for your time looking at it.

Reply:
👍 @frankhereford and i defer to whatever y'all think is the right amount of investment on this. just wanted to make sure you had the feedback. thanks for taking it.